src/main/java/com/eod/eod/domain/introduce/application/IntroduceService.java

```diff
@@ -0,0 +1,33 @@
+package com.eod.eod.domain.introduce.application;
+
+import com.eod.eod.domain.introduce.infrastructure.IntroduceRepository;
+import com.eod.eod.domain.introduce.model.Introduce;
+import com.eod.eod.domain.introduce.presentation.dto.IntroduceUpdateResponse;
+import com.eod.eod.domain.user.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class IntroduceService {
+
+    private final IntroduceRepository introduceRepository;
+
+    @Transactional
+    public IntroduceUpdateResponse updateIntroduce(String content, User currentUser) {
+        // 가장 최근 소개 페이지 조회 또는 새로 생성
+        Introduce introduce = introduceRepository.findFirstByOrderByIdDesc()
```

Comment:
이 부분을 이렇게 처리해주시면 id가 null일 경우가 없어져 따로 Null 처리 안해주셔도 될 듯 합니다.
```suggestion
        Introduce introduce = introduceRepository.findFirstByOrderByIdDesc()
            .orElseGet(() -> introduceRepository.save(new Introduce(content)));
```